### ISSUE TITLE
aria-describedby with angular material with custom error messages

### ISSUE THREAD

**Issue by MarieCharbonneau2** [ngx-formly/ngx-formly#1985]
I'm submitting a ... (check one with "x")

[ ] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[x] support request

Current behavior
I use material fields and custom error messages.
I want to use the attribute aria-describedby but it doesn't seem to work.

Please tell us about your environment:
Angular 7, Material

You can see a reproduction example here : https://stackblitz.com/edit/angular-ztgoco

Thanks

**Comment by aitboudad** [comment]
use `attributes`:

```ts
templateOptions: {
  attributes: {
    'labelledby': 'label-mail'
  }
},
```

**Comment by MarieCharbonneau2** [comment]
the problem is not for labelledby but for describedby.
It does not work nether with attributes

**Comment by MarieCharbonneau2** [comment]
the problem is not for labelledby but for describedby

**Comment by MarieCharbonneau2** [comment]
Thanks. But it does not work with the atribute aria-describedby

**Comment by aitboudad** [comment]
Well there is no way to set a custom `describedby` for material input: https://github.com/angular/components/blob/master/src/material/input/input.ts#L83
it all controlled by `MatFormFieldControl`.
